src/renderer/vt/XtermEngine.cpp

```diff
@@ -408,9 +408,10 @@ XtermEngine::XtermEngine(_In_ wil::unique_hfile hPipe,
 
     if (dx != 0 || dy != 0)
     {
-        // Scroll the current offset
-        RETURN_IF_FAILED(_InvalidOffset(pcoordDelta));
+        // Scroll the current offset and invalidate the revealed area
+        _invalidMap.translate(til::point(*pcoordDelta), true);
 
+        // TODO: MIGRIE DO WE NEED THIS IF THE TRANSLATION ABOVE FILLS THE VACATED AREA?

```

Comment:
Sure doesn't seem like we need it anymore

Reply:
K, I'll pull it and make sure the tests still pass. If so, then we'll call it good enough for now.